plugins/parsers/wavefront/parser.go

```diff
@@ -47,9 +53,41 @@ func NewWavefrontElements() []ElementParser {
 	return elements
 }
 
-func NewWavefrontParser(defaultTags map[string]string) *PointParser {
+func NewWavefrontParser(defaultTags map[string]string) *WavefrontParser {
+	wp := &WavefrontParser{defaultTags: defaultTags}
+	wp.parsers = &sync.Pool{
+		New: func() interface{} {
+			log.Printf("D! [parsers.wavefront] Creating new parser for %s", wp)
```

Comment:
(This is the ci test failure) I would just remove the `for %s` part completely.  Default tags never change unless Telegraf is reloaded and are shared on all inputs.

Reply:
Done!